bin/run.js

```diff
@@ -602,6 +604,17 @@ const runAllFiles = (options, tap) => {
       if (options.jobs > 1)
         opt.buffered = isParallelOk(parallelOk, file) !== false
 
+      if(options.flow){
+        debug('flow', file)
+        const args = [
+          ...(options['node-arg']),
+          flowNode,
+          file,
+          ...(options['test-arg']),
+        ]
+        tap.spawn(node, args, opt, file)
```

Comment:
This will double-spawn in most cases, I think.  (Ie, here, and then down below when it detects that it's a *.js file.)

Rather than spawn here, I think you can just `options['node-arg'].push('-r', flowNode)` if `--flow` is set, and then let the spawn be handled below.

Reply:
I see how that works now. Sorry about that.